repo/docs/Single_Sign_On.md

```diff
@@ -0,0 +1,87 @@
+# Single Sign On (SSO) Integration
+
+- [OIDC](#oidc) 
+- [SAML](#saml)
+
+## Introduction
+
+SSO is supported via [OIDC](https://openid.net/developers/how-connect-works/) or [SAML](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language).
+
+## <a name="oidc"></a>OIDC
+
+### Configuration
+
+Supply a map as below.
+
+```yaml
+global:
+  conf:
+    sso:
```

Comment:
Didn't notice this ... let's lose the `global.conf.sso` middle step, and go straight to `global.sso`. `global.conf.XXX` is meant (with one notable exception - `baseUrl`) to have the `XXX` key represent the name of a given subsystem (i.e. `reports`, `ldap`, etc... for further configuration.